Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:18
+
+WORKDIR /app
+
+COPY package*.json ./
+
+RUN npm install
+
+COPY . .
+
```

Comment:
When creating a Dockerfile to prepare an app for production, you would also build the app within the container. 
For development env, just running the app as-is is enough so this Dockerfile is fine.
But let's take the chance and improve it? Add the build step as well.

Reply:
You are also missing the instructions in the Readme.